docker-compose.yml

```diff
@@ -0,0 +1,38 @@
+version: "3.9"
+services:
+  monitor:
+    build:
+      context: .
+      dockerfile: Dockerfile.monitor
+    ports:
+      - "3001:3001"
+    volumes:
+      - ~/config:/oracle-cl-middleware/src/config
+      - ~/state:/oracle-cl-middleware/src/data
+    environment:
+      AGORIC_NET: agoriclocal 
+      AGORIC_RPC: http://$THIS_VM_IP:26657
+      EI_CHAINLINKURL: http://$THIS_VM_IP:6691
```

Comment:
Not used in monitor.js

Reply:
Fixed in https://github.com/jacquesvcritien/agoric-cl-middleware/commit/0f2d4ec7dd638efe9fb0c21d9d11c8de0c3fb01e

Reply:
Fixed